### DISCUSSION TITLE
Adding flag to copy documentation to given location

### DISCUSSION THREAD

**Discussion by LiisiKerik** [haskell/cabal#8724] [Polls]
I created this issue a few months ago: https://github.com/haskell/cabal/issues/8270

I would like to be able to redirect `cabal haddock` output to a folder of my choice. In comments it was suggested to add an `--only-documentation` flag to `cabal install`.

I've been trying to implement this change and realised that I don't actually know what would be the most desirable behavior for this command.

Should it only copy documentation that's already been generated? Or should it generate documentation? If it generates documentation, then should it be also able to re-run haddock and overwrite the existing documentation even if documentation already exists?

If it's able to run haddock (and not just copy documentation) then should it put the documentation in the standard folder first and then make a copy? Or redirect the documentation where the user wants it and create just this one copy?

Is adding this option to `cabal install` the most reasonable way to do this? Are there other, maybe better ways of achieving the same end? Wouldn't it be better to add some kind of additional flag / argument to `cabal haddock`?

I also made a reddit thread about this same topic: https://www.reddit.com/r/haskellquestions/comments/vi0tcm/how_to_redirect_cabal_haddock_output_to_a/

**Comment by ulysses4ever** [comment]
I believe this was divided in the bug tracker. Someone, please, try to find it. I'm the meantime, closing as outdated.
